integreat_cms/cms/models/abstract_tree_node.py

```diff
@@ -282,7 +287,21 @@ def move(self, target, pos=None):
         # Moving a node can modify all other nodes via raw sql queries (which are not recognized by cachalot),
         # so we have to invalidate the whole model manually.
         invalidate_model(self.__class__)
-        super().move(target=target, pos=pos)
+        try:
+            with db_mutex(f"tree-page-{self.region}"):
+                super().move(target, pos)
+        except DBMutexError as e:
+            logger.error("Could not retrieve database mutex to save %s", self)
+            raise DBMutexError(
+                # pylint: disable=consider-using-f-string
+                _('Could not change position in tree of "{}".'.format(self))
```

Comment:
```suggestion
                _('Could not change position in tree of "{}".').format(self)
```
You need to translate first, then perform the string formatting.

Reply:
:facepalm: